babel-intro/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "babel-intro",
+  "version": "1.0.0",
+  "description": "",
+  "main": "webpack.config.js",
```

Comment:
fyi: we want to create our json package first by running "npm init -y" before installing the dependencies. 
That will result in  "main":"index.js".